go/lambda/ConfigureFunction/ConfigureFunction.go

```diff
@@ -0,0 +1,69 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// snippet-start:[lambda.go.config_function]
+package main
+
+// snippet-start:[lambda.go.config_function.imports]
+import (
+    "flag"
+    "fmt"
+
+    "github.com/aws/aws-sdk-go/aws"
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/lambda"
+    "github.com/aws/aws-sdk-go/service/lambda/lambdaiface"
+)
+// snippet-end:[lambda.go.config_function.imports]
+
+// AddPerm enables an Amazon S3 bucket to send notifications to a Lambda function.
+// Inputs:
+//     svc is a Lambda service client
+//     function is the name of the Lambda function
+//     s3ARN is the ARN of the Amazon S3 bucket
```

Comment:
...is the **Amazon Resource Name (ARN)** of...

Reply:
Fixed